WordPressKit/WordPressKit/Activity.swift

```diff
@@ -31,8 +31,7 @@ public class Activity {
         guard let publishedString = dictionary["published"] as? String else {
             throw Error.missingPublishedDate
         }
-        let dateFormatter = ISO8601DateFormatter()
-        guard let publishedDate = dateFormatter.date(from: publishedString) else {
+        guard let publishedDate = Date.dateWithISO8601WithMillisecondsString(publishedString) else {
```

Comment:
oh no! I was hoping this would parse without needing any other changes.

one thing I think might be valuable here is indicating "fractional seconds" or something like that instead of "milliseconds" because it's also plausible that in the future we send with microseconds. this isn't because we want to change things, but I've noticed that many systems _either_ produce no decimal seconds _or_ they send microseconds.

it's just a name right now so no big deal, but just some comment on the name…

Reply:
Well but currently we are sending milliseconds right? If so I want to keep this name since we might use the formatter in other places of the app.